src/main/java/org/broadinstitute/hellbender/utils/bigquery/BigQueryUtils.java

```diff
@@ -1,495 +0,0 @@
-package org.broadinstitute.hellbender.utils.bigquery;
-
-import com.google.cloud.bigquery.*;
-import io.grpc.StatusRuntimeException;
-import org.apache.ivy.util.StringUtils;
-import org.apache.logging.log4j.LogManager;
-import org.apache.logging.log4j.Logger;
-import org.broadinstitute.hellbender.exceptions.GATKException;
-
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Map;
-import java.util.UUID;
-import java.util.stream.Collectors;
-import java.util.stream.IntStream;
-import java.util.stream.StreamSupport;
-
-/**
- * Utility class for dealing with BigQuery connections / tables / queries /etc.
- *
- * Created by jonn on 4/17/19.
- */
-public final class BigQueryUtils {
```

Comment:
Instead of deleting and re-adding these, it might be better to simply move them to their destined package in this PR. That way, we could preserve the git history of changes to these utils.

Reply:
This was done so I'm going to dismiss this review.